app/models/event.rb

```diff
@@ -0,0 +1,6 @@
+class Event < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
Oh, actually I think we had this discussion in the Łódź office. Sorry I didn't tell you.